workflow/personal_genome_input_wf.cwl

```diff
@@ -0,0 +1,80 @@
+cwlVersion: v1.2
+class: Workflow
+id: personal-genome-input-wf
+label: Personal Genome Input Workflow
+doc: "Filter an input VCF and remove annotations"
+
+inputs:
+  # Strip, subset and PASS vars
+  input_vcf: {type: File, secondaryFiles: ['.tbi']}
+  strip_info: {type: 'string?', doc: "If given, remove previous annotation information based on INFO file, i.e. to strip VEP info, use INFO/ANN"}
+  output_basename: string
+  include_expression: { type: 'string?', doc: "See bcftools docs for valid expression. Can't be used at the same time as exclude_expression"}
+  exclude_expression: {type: 'string?', doc: "See bcftools docs for valid expression. Can't be used at the same time as include_expression"}
+  filter_type: { type: 'string?', doc: "Apply a FILTER value expression", default: "PASS"}
+  subtract_bed: {type: 'File?', doc: "Supply if you want to remove regions for any reason, like low complexity or repeat mask, etc" }
+  # Genome gen vars
+  genomeDir: { type: string, doc: "Output dirname. Recommend STAR_{version}_GENCODE{version num}_{Patient/sample id}" }
```

Comment:
Seems dubious to allow people to manipulate the file system. Shouldn't this just be `output_basename` anyway?